model/event/distro_event.go

```diff
@@ -57,8 +61,13 @@ func LogDistroAdded(distroId, userId string, data interface{}) {
 }
 
 // LogDistroModified should take in DistroData in order to preserve the ProviderSettingsList
-func LogDistroModified(distroId, userId string, data interface{}) {
-	LogDistroEvent(distroId, EventDistroModified, DistroEventData{UserId: userId, Data: data})
+func LogDistroModified(distroId, userId string, before, after interface{}) {
+	// Stop if there are no changes
+	if reflect.DeepEqual(before, after) {
+		return
+	}
+
+	LogDistroEvent(distroId, EventDistroModified, DistroEventData{UserId: userId, User: userId, Data: after, Before: before, After: after})
```

Comment:
nit: preference to construct the event data on the previous line to break this line down a bit